modules/shared/src/test/scala/io/constellationnetwork/security/HashSuite.scala

```diff
@@ -58,6 +60,21 @@ object HashSuite extends MutableIOSuite with Checkers {
     res.withCurrent(implicit hasher => (hasher.hash(test), hasher.hash(testUpdated)).mapN(expect.eql(_, _)))
   }
 
+  test("ensure guava/JSA compatibility") {
+    val data = Array.ofDim[Byte](1024)
+
+    val (guavaHashes, jsaHashes) = 0.to(1_000_000).foldLeft((Vector.empty[String], Vector.empty[String])) { (acc, _) =>
+      val (guava, jsa) = acc
+      scala.util.Random.nextBytes(data)
+      @nowarn
+      val hashCode = Hash.hashCodeFromBytes(data)
+      val sha256Digest = Hash.sha256DigestFromBytes(data)
+      (guava :+ hashCode.toString, jsa :+ sha256Digest.toHexString)
+    }
+
+    IO(expect(guavaHashes == jsaHashes))
```

Comment:
Expect should use Eq (===) from cats, instead of the universal equality. Also, there is also expect same or expect equal functions that would work in your case

Reply:
Done